[2025-09-16T15:55:35.732+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_pipeline_postgres.transform manual__2025-09-16T15:55:30.524755+00:00 [queued]>
[2025-09-16T15:55:35.738+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_pipeline_postgres.transform manual__2025-09-16T15:55:30.524755+00:00 [queued]>
[2025-09-16T15:55:35.738+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-09-16T15:55:35.746+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): transform> on 2025-09-16 15:55:30.524755+00:00
[2025-09-16T15:55:35.750+0000] {standard_task_runner.py:57} INFO - Started process 173 to run task
[2025-09-16T15:55:35.752+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'crypto_pipeline_postgres', 'transform', 'manual__2025-09-16T15:55:30.524755+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/coleta_dados.py', '--cfg-path', '/tmp/tmpljc10d73']
[2025-09-16T15:55:35.753+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask transform
[2025-09-16T15:55:35.782+0000] {task_command.py:415} INFO - Running <TaskInstance: crypto_pipeline_postgres.transform manual__2025-09-16T15:55:30.524755+00:00 [running]> on host f0ba1f59a373
[2025-09-16T15:55:35.825+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_pipeline_postgres' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-09-16T15:55:30.524755+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-16T15:55:30.524755+00:00'
[2025-09-16T15:55:35.833+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-09-16T15:55:35.839+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=crypto_pipeline_postgres, task_id=transform, execution_date=20250916T155530, start_date=20250916T155535, end_date=20250916T155535
[2025-09-16T15:55:35.884+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-16T15:55:35.898+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
